.github/workflows/release.yml

```diff
@@ -1,19 +1,31 @@
 name: Release
-
 on:
-  workflow_dispatch:
   push:
     tags:
-      - 'v*'
-
+      - 'v*.*.*'
 jobs:
-  homebrew:
-    runs-on: macos-latest
+  build-binaries:
+    uses: ./.github/workflows/build-binaries.yml
+
+  create-release:
+    runs-on: ubuntu-latest
+    needs: [build-binaries]
     steps:
-      - name: Checkout repo
+      - name: Checkout repository
```

Comment:
Why do you need to check out the repo here?

Reply:
it was in the old release action. Possibly because of the homebrew formula?

Reply:
true, looks like it doesn't need it: https://github.com/dawidd6/action-homebrew-bump-formula?tab=readme-ov-file#standard-mode

Reply:
I was about to check it tonight, but you were faster ;).